nodebuilder/core/config.go

```diff
@@ -15,6 +15,13 @@ var MetricsEnabled bool
 
 // Config combines all configuration fields for managing the relationship with a Core node.
 type Config struct {
+	EndpointConfig
```

Comment:
Original endpoint could become a slice or is the idea to not break the config?

Reply:
idea was not to break config but to make it additive change - og endpoint cfg would be untouched.